src/state/preferences/hide-follow-notifications.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+
+import * as persisted from '#/state/persisted'
+
+type StateContext = persisted.Schema['hideFollowNotifications']
+type SetContext = (v: persisted.Schema['hideFollowNotifications']) => void
+
+const stateContext = React.createContext<StateContext>(
+  persisted.defaults.hideFollowNotifications,
```

Comment:
you use the default here but it'll be undefined I think?

Reply:
Oops, forgot about this. Will do.